doc/howto/using_the_api.rst

```diff
@@ -41,6 +41,21 @@ on NAV installations that do not have SSL enabled, you are potentially giving
 everyone access to the data.
 
 
+JSON Web Tokens
+------------------
+JSON Web Tokens (JWTs) must be :doc:`configured <../reference/jwt>` before they can be used.
+
+Once configured, you can use tokens issued by your configured issuers in almost the same way
+as :ref:`classic tokens <classic-token>`::
+
+  curl -H 'Authorization: Bearer <token>' https://<host>/api/netboxes/
+
+Note how JWTs require the prefix `Bearer` and not `Token`.
+
+JWTs must include valid `exp`, `nbf`, `iss` and `aud` claims in order to be valid.
+`ìss` and `aud` must match the :doc:`configuration <../reference/jwt>`, while `exp` must
+be in the future and `nbf` must be in the past.
```

Comment:
Remember, this is rST, not Markdown.  There is a semantic difference between single and double backticks, and you usually want double backticks in this case to denote "inline literals", which should be rendered as monospace text:

```suggestion
Note how JWTs require the prefix ``Bearer`` and not ``Token``.

JWTs must include valid ``exp``, ``nbf``, ``iss`` and ``aud`` claims in order to be valid.
``iss`` and ``aud`` must match the :doc:`configuration <../reference/jwt>`, while ``exp`` must
be in the future and ``nbf`` must be in the past.
```